CHANGELOG.md

```diff
@@ -5,30 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [0.12.0]
+## [1.1.0]
```

Comment:
Why is this 1.1.0 and not 1.0.0?

Reply:
there is 1.0.0 already in legacy already https://github.com/giantswarm/kubernetes-nginx-ingress-controller/blob/master/CHANGELOG.md#100

Reply:
@corest Ack because of the breaking change in 0.26.1. I remember now. 😅